### PR TITLE
Limit how many files can be changed in a PR

### PR DESCRIPTION
**Pull request by iarspider**
Redo of #2196

### CODE REVIEW

**Comment by iarspider** [comment]
hold

**Comment by smuzaffar** [comment]
@iarspider , is this ready to go in now?

**Comment by iarspider** [comment]
yes

File: process_pr.py [hunk 1]
```diff
@@ -1549,6 +1580,39 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
             print("Commit count reached and not overridden, quitting")
             return
 
+        if (
+            pr.changed_files >= TOO_MANY_FILES_WARN_THRESHOLD
+            and (not warned_too_many_files)
+            and (not ok_too_many_files)
+        ):
+            if pr.commits < TOO_MANY_FILES_FAIL_THRESHOLD:
```
Comment (smuzaffar) [inline process_pr.py:1588]: @iarspider , I think this should be `pr.changed_files`

**Comment by smuzaffar** [comment]
ping bot

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**